README.md

```diff
@@ -428,6 +428,48 @@ in Step 1 of the
 [Set up a Conjur OSS environment with persistence](#set-up-a-conjur-oss-environment-with-persistence)
 section above.
 
+### Adding or Modifying Container Environment Variables
+
+This section describes the process of either adding or modifying  environment variables in a running container. This process only requires stopping the container in question for a short time, while the rest of the system continues running.
```

Comment:
> This section describes the process of either adding or modifying  environment variables in a running container. This process only requires stopping the container in question for a short time, while the rest of the system continues running.

I think this paragraph is a bit inaccurate since the container will not be running anymore once you run the steps below, it gets altogether destroyed and a new one is created after it. For stateless containers (e.g. Conjur) we don't mind when they get destroyed because they are stateless (Conjur's backing store is the database service which would not interrupted when these steps for the Conjur service). For stateful containers (e.g. the database service) we might mind because if our state is not persisted then upon recreation all our data is gone.

I wonder if we can rephrase the above to something like:
> This section describes the process of either adding or modifying environment variables for a `docker-compose` service. The process recreates the desired service so that the changes may be reflected, while the rest of the system continues running. Note that for stateful service like the database you must have a persistence mechanism in place (e.g. volume mounts) otherwise when the container is recreated you'll lose all your data.